### DISCUSSION TITLE
Numbers for the Graph?

### DISCUSSION THREAD

**Discussion by syncronox** [merchantprotocol/theadslab#35] [Open Discussion Forum]
I want to know the numbers for this graph so I can explain to prospects how it compares to competitors.

Specifically:
How many people
How many emails, phone numbers and addresses
How many points of data

Last seen date? They filled out an application on this date with this phone number

**Comment by jonathonbyrdziak** [comment]
Here are the numbers as of today

| Description | Count |
|--|--|
| Average number of **connected** devices in the United States | 10.7 Billion |
| The number of known devices in our database | 22 Billion |
| The amount of connected devices we're able to identify | 40% |
| The number of unique emails in our database | 4.7 Billion |
| The number of email addresses we're able to match to devices we identify | currently 45%, will be 95% by June 2022 |
| Number of citizens in the United States | 331 Million |
| Number of citizen profiles we track | 266 Million |
| Max number of data points per profile record | 5,700 |

## Categories tracked 
Demographic, Geographic, Psychographic, Affinities, Buying Habits, Finances, Automobiles, Motorcycles, Boats, Auto Insurance, Life Insurance, Health Insurance, Social profiles, Political affiliation, etc 

The "last seen" was when we saw them on your website last.

For each record type we do have dates associated with each account that shows when we last saw or captured the data. It's different for every record and our database has a mass update 4 times per year. Along with thousands of micro updates throughout the year.

**Reply by syncronox** [reply]
I have a question-How are there so many unique emails and known devices in the database but only 266 million people? 266 million people is a lot but 10 billion devices and 4.7 billion emails? Do they all belong to these 266 million people? I'm a little confused about that.

Also it says there are 5,700 data points for each profile record, I only see 658 parameters listed so I am a little confused about that.
